app/src/main/java/com/nextcloud/talk/controllers/ContactsController.java

```diff
@@ -658,12 +665,22 @@ private void prepareViews() {
             }
         });
 
+        joinConversationViaLinkImageView
+                .getBackground()
+                .setColorFilter(ResourcesCompat.getColor(getResources(), R.color.colorPrimary, null),
```

Comment:
Should use the grey (server name: `color-background-darker`) which is `#dbdbdb` in bright and `#2c2c2c`  in dark mode

Reply:
DONE

Reply:
https://github.com/nextcloud/talk-android/pull/1209/commits/79e5c3bcaff7e18050da381a125a54a17a9bcb92